Dockerfile

```diff
@@ -16,17 +16,30 @@ RUN apt-get update && \
     sudo apt-get install -y -q python3-pylint-django && \
     sudo apt-get install -y -q pycodestyle && \
     sudo apt-get install -y -q swig
+RUN sudo apt-get install -y -q python3-pip
 
 ADD . /app
 
 WORKDIR /app/embark
 
 ADD embark/requirements.txt /app/embark/requirements.txt
 
-RUN yes | sudo /app/emba/installer.sh -D  && \
-    sudo pip3 install uwsgi -I --no-cache-dir && \
+RUN yes | sudo pip3 install uwsgi -I --no-cache-dir && \
```

Comment:
With this change we have no EMBA installation within the EMBArk container anymore. This is needed as scanning backend. Do I miss something?

Reply:
Right now embark is running the emba-analysis in Developer mode.

![image](https://user-images.githubusercontent.com/62940240/139806580-5da07936-2635-48f1-897d-b6cd942980f4.png)

Reply:
EMBA is running within the docker container of EMBArk in dev mode. With this PR the installation process of EMBA will be removed and will break the complete system.